client/src/main/java/com/sanction/thunder/ThunderBuilder.java

```diff
@@ -17,8 +17,9 @@
    * @param endpoint  The base URL of the API endpoint to connect to.
    * @param apiUser   The API username to use when connecting to the endpoint.
    * @param apiSecret The API secret to use when connecting to the endpoint.
+   * @param apiPassword The API password to use when connecting to the endpoint.
    */
-  public ThunderBuilder(String endpoint, String apiUser, String apiSecret) {
+  public ThunderBuilder(String endpoint, String apiUser, String apiSecret, String apiPassword) {
```

Comment:
Okay is isn't right, because the `ThunderBuilder` is only created once. This password is per user and has to be changed on each request.

We actually don't need this here, as long as we have it in the parameter list in the `ThunderClient` interface then it will get send over as a header.

So, this class should not have any changes at all.